### ISSUE TITLE
[BUG] Unhandled rejection when running script

### ISSUE THREAD

**Issue by jagomf** [compodoc/compodoc#669]
##### **Overview of the issue**

I run the script with
`compodoc -p src/tsconfig.app.json -s --theme material`

then during execution, and after successfully parsing several files, I get
```
Unhandled Rejection at: Promise {
  <rejected> Error: Expected to find an initializer.
    at InvalidOperationError.BaseError [as constructor] ([project]\node_modules\ts-simple-ast\dist\errors\BaseError.js:7:28)
    at new InvalidOperationError ([project]\node_modules\ts-simple-ast\dist\errors\InvalidOperationError.js:8:28)
    at Object.throwIfNullOrUndefined ([project]\node_modules\ts-simple-ast\dist\errors\helpers.js:96:15)
    at EnumMember.InitializerGetExpressionableNode.class_1.getInitializerOrThrow ([project]\node_modules\ts-simple-ast\dist\compiler\base\initializer\InitializerGetExpressionableNode.js:25:27)
    at RouterParserUtil.cleanFileDynamics ([project]\node_modules\@compodoc\compodoc\dist\index-cli.js:1381:80)
    at AngularDependencies.getSourceFileDecorators ([project]\node_modules\@compodoc\compodoc\dist\index-cli.js:6306:45)
    at [project]\node_modules\@compodoc\compodoc\dist\index-cli.js:6118:31
    at Array.map (<anonymous>)
    at AngularDependencies.getDependencies ([project]\node_modules\@compodoc\compodoc\dist\index-cli.js:6106:21)
    at CliApplication.Application.getDependenciesData ([project]\node_modules\@compodoc\compodoc\dist\index-cli.js:7971:40) } reason: Error: Expected to find an initializer.
    at InvalidOperationError.BaseError [as constructor] ([project]\node_modules\ts-simple-ast\dist\errors\BaseError.js:7:28)
    at new InvalidOperationError ([project]\node_modules\ts-simple-ast\dist\errors\InvalidOperationError.js:8:28)
    at Object.throwIfNullOrUndefined ([project]\node_modules\ts-simple-ast\dist\errors\helpers.js:96:15)
    at EnumMember.InitializerGetExpressionableNode.class_1.getInitializerOrThrow ([project]\node_modules\ts-simple-ast\dist\compiler\base\initializer\InitializerGetExpressionableNode.js:25:27)
    at RouterParserUtil.cleanFileDynamics ([project]\node_modules\@compodoc\compodoc\dist\index-cli.js:1381:80)
    at AngularDependencies.getSourceFileDecorators ([project]\node_modules\@compodoc\compodoc\dist\index-cli.js:6306:45)
    at [project]\node_modules\@compodoc\compodoc\dist\index-cli.js:6118:31
    at Array.map (<anonymous>)
    at AngularDependencies.getDependencies ([project]\node_modules\@compodoc\compodoc\dist\index-cli.js:6106:21)
    at CliApplication.Application.getDependenciesData ([project]\node_modules\@compodoc\compodoc\dist\index-cli.js:7971:40)
```

Note: I have replaced full project path with `[project]`.

##### **Operating System, Node.js, npm, compodoc version(s)**

OS: Windows 10, x64 (using PowerShell)
Compodoc: 1.1.5
Node: 8.9.4
NPM: 5.6.0
Angular: 6.0.3

##### **Angular configuration, a `package.json` file in the root folder**

I'm not sure wether you need angular.json or package.json...?

##### **Compodoc installed globally or locally ?**

Installed compodoc locally to project with `npm i --save-dev @compodoc/compodoc`

##### **If possible your terminal logs before the error**

Last log before error:
```
parsing: [project]/src/app/groups/groups-routing.module.ts
Analysing routes definitions and clean them if necessary
```
This is the content of the file `groups-routing.module.ts`:
```typescript
import { NgModule } from '@angular/core';
import { RouterModule, Routes } from '@angular/router';
import { GroupHandlerComponent } from './handlers/group-handler/group-handler.component';
import { ResolveGroup, GroupsService } from './services/groups.service';

const routes: Routes = [
	{
		path: '',
		component: GroupHandlerComponent,
		resolve: { content: GroupsService },
		data: { needs: ResolveGroup.multiple }
	},
	{
		path: ':groupid',
		component: GroupHandlerComponent,
		resolve: { content: GroupsService },
		data: { needs: ResolveGroup.single }
	}
];

@NgModule({
	imports: [RouterModule.forChild(routes)],
	exports: [RouterModule]
})
export class GroupsRoutingModule {}

```

##### **Reproduce the error**

Run `compodoc -p src/tsconfig.app.json -s --theme material`

**Comment by jagomf** [comment]
Seems to work without the `data` attribute in the `Routes` object!

**Comment by seydimomogueye** [comment]
Hi 
I had the same issue, i just removed the typing (**export const routes: Routes = []** to **export const routes = []**) and it's worked

**Comment by vogloblinsky** [comment]
Fixed since 1.1.14
